CMakeLists.txt

```diff
@@ -88,9 +105,17 @@ if (MILVUS_SUPPORT_SPTAG)
     add_compile_definitions("MILVUS_SUPPORT_SPTAG")
 endif ()
 
-include(ThirdPartyPackagesCore)
-add_subdirectory(knowhere/knowhere/utils)
-add_subdirectory(thirdparty)
+add_subdirectory(knowhere)
+if (MACOS)
+    include_directories(${INDEX_SOURCE_DIR}/thirdparty/easyloggingpp)
+    include_directories(${INDEX_SOURCE_DIR}/thirdparty/nolohmann)
+    add_subdirectory(selfdev/bruteforce)
+endif ()
+
+if (LINUX)
```

Comment:
use elif ()

Reply:
already use elif() in 40 lines,  so if() more clear,  elif() should be used with else() together